src/main/java/com/google/codeu/data/Message.java

```diff
@@ -30,15 +32,18 @@
    * Constructs a new {@link Message} posted by {@code user} with {@code text} content. Generates a
    * random ID and uses the current system time for the creation time.
    */
-  public Message(String user, String text) {
-    this(UUID.randomUUID(), user, text, System.currentTimeMillis());
+  //add recipient to the constructor
+  public Message(String user, String text, String recipient) {
```

Comment:
Update the constructor comment to include @param javadocs.

Reply:
Added.